### DISCUSSION TITLE
Combined Results

### DISCUSSION THREAD

**Discussion by twharmon** [krausest/js-framework-benchmark#1526] [Ideas]
Could there be a results table that combines the duration, memory, and size results into an overall score? I expect different people place different weight on the 3 measurements when evaluating frameworks, so maybe they could edit the weight in place in the far left column?

**Comment by leeoniya** [comment]
personally, i would find a composite score of completely different things more confusing than helpful, even if i could custom weight each factor. you can already sort by whatever factor is most important and then scan the other tables for secondary and tertiary factors.

/$0.02
